### PR TITLE
MM-30370: Cypress/E2E: Automate backlogs - Keyboard Shortcuts (10 test cases)

### PR DESCRIPTION
**Pull request by catalintomai**
#### Summary
Cypress/E2E: Automate backlogs - Keyboard Shortcuts (10 test cases)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-30370

### CODE REVIEW

**Review by DHaussermann — APPROVED** [review]
Tested and passed 
- Test actions are correct
- All test run Green
- Updated Cypress status on all corresponding TM4J tests

LGTM!

File: e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_one_spec.js [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+        // # Login as admin and visit town-square
+        cy.apiAdminLogin();
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+    });
+
+    it('MM-T1224 - CTRL/CMD+K - Open DM using mouse', () => {
+        // # Type CTRL/CMD+K
+        cy.get('#post_textbox').cmdOrCtrlShortcut('K');
+
+        cy.apiCreateUser({prefix: 'taemp-'}).then(({user: tempUser}) => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_one_spec.js:46]:
```suggestion
        cy.apiCreateUser({prefix: 'temp-'}).then(({user: tempUser}) => {
```

**Review by saturninoabril — APPROVED** [review]
Thanks Catalin, really great!

**Pull request merged**